sql/insert.go

```diff
@@ -19,15 +19,34 @@ package sql
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/cockroachdb/cockroach/client"
 	"github.com/cockroachdb/cockroach/proto"
 	"github.com/cockroachdb/cockroach/sql/parser"
 	"github.com/cockroachdb/cockroach/sql/privilege"
-	"github.com/cockroachdb/cockroach/util"
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
+type writeMap map[*ColumnDescriptor]parser.Datum
+
+func (wm writeMap) String() string {
+	cols := make([]string, 0, len(wm))
+	vals := make([]string, 0, len(wm))
+
+	for col, val := range wm {
```

Comment:
Map traversal is randomized. Probably best to make the output of `writeMap.String` deterministic.